msgid "cceErrorSentence"
msgstr "Leider trat ein interner Serverfehler auf."

msgid "cceHandlerErrorSentence"
msgstr "Leider trat ein interner Serverfehler auf. [[base-sauce-basic.contactTechSupport]]"

msgid "unknownAttr"
msgstr "[[base-cce.cceErrorSentence]]  Das Attribut [[VAR.key]] ist unbekannt. [[base-sauce-basic.contactTechSupport]]"

msgid "unknownClass"
msgstr "[[base-cce.cceErrorSentence]]  Die Klasse [[VAR.key]] ist unbekannt. [[base-sauce-basic.contactTechSupport]]"

msgid "unknownNs"
msgstr "[[base-cce.cceErrorSentence]]  Der Namespace [[VAR.key]] ist unbekannt. [[base-sauce-basic.contactTechSupport]]"

msgid "unknownOid"
msgstr "[[base-cce.cceErrorSentence]]  Das Objekt [[VAR.oid]] ist unbekannt. [[base-sauce-basic.contactTechSupport]]"

msgid "permissionDenied"
msgstr "[[base-cce.cceErrorSentence]]  Berechtigung für [[VAR.key]] wird verweigert. [[base-sauce-basic.contactTechSupport]]"

msgid "unknownErrorCode"
msgstr "[[base-cce.cceErrorSentence]]  Ein unbekannter Fehler trat auf. [[base-sauce-basic.contactTechSupport]]"

msgid "invalidType"
msgstr "[[base-cce.cceErrorSentence]]  Der Typ [[VAR.key]] ist ungültig. [[base-sauce-basic.contactTechSupport]]"

	# User input invalid but validation fell through
msgid "invalidData"
msgstr "Die in das Feld [[VAR.key]] eingegebenen Daten sind leider ungültig. Bitte überprüfen Sie Ihre Eingabe und versuchen Sie es noch einmal."

msgid "outOfMemory"
msgstr "[[base-cce.cceErrorSentence]]  Das System kann nicht genug Speicher zuweisen, um fortzufahren."
